sync.py

```diff
@@ -247,11 +251,11 @@ def main():
     parser.add_argument(
         'action',
         choices={'startserver', 'sync'},
-        help='use "startserver" to start the server and "sync --repo [name]" to force a sync',
+        help='use "startserver" to start the server and "sync --repo [name]" to force a meta-repo sync',
     )
-    parser.add_argument('--verbose', '-v', action='count', help='adjust verbosity', default=0)
+    parser.add_argument('--verbose', '-v', action='count', help='add verbosity (maximum -v -v)', default=0)
```

Comment:
okay, so, the maximum should be -vvv. this is a bug that could use fixing

Reply:
`levels = [logging.WARNING, logging.INFO, logging.DEBUG]`
Which option count corresponds to which level?

Reply:
I was thinking -v, -vv and -vvv respectively

Reply:
And if no -v is given?

Reply:
default I suppose?